packages/app/components/TokenIcon.tsx

```diff
@@ -5,31 +5,45 @@ import { useNetwork } from "wagmi";
 import { twMerge } from "tailwind-merge";
 import { useTokenListContext } from "@/context/TokenListContext";
 import { Token } from "@/models/token/types";
+import { useState } from "react";
 
 interface TokenIconProps {
-  token: Token;
   className?: string;
   size?: "lg" | "md" | "sm" | "xs";
+  token: Token;
 }
 
-export const TokenIcon = ({ token, className, size }: TokenIconProps) => {
+export const TokenIcon = ({
+  className,
+  size = "sm",
```

Comment:
we don't need to add a default arg since the `cva` has a `defaultVariants` (`sm`)

Reply:
Ohhh I see so CVA handles those default values, uhm... I'll bear that in mind, thanks! WIll update it now

Reply:
yup!